addons/binding/org.openhab.binding.sensebox/src/main/java/org/openhab/binding/sensebox/internal/SenseBoxHandlerFactory.java

```diff
@@ -19,15 +19,14 @@
 import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
 import org.eclipse.smarthome.core.thing.binding.ThingHandler;
 import org.osgi.service.component.annotations.Component;
-import org.osgi.service.component.annotations.Reference;
 
 /**
  * The {@link SenseBoxHandlerFactory} is responsible for creating things and thing
  * handlers.
  *
  * @author Hakan Tandogan - Initial contribution
  */
-@Component(service = BaseThingHandlerFactory.class, immediate = true, name="binding.sensebox")
+@Component(service = BaseThingHandlerFactory.class, immediate = true, configurationPid = "binding.sensebox")
```

Comment:
@hakan42 I just notice that I didn't check your first PR enough. As you can see [here](https://github.com/eclipse/smarthome/pull/4113/files), the service must be `ThingHandlerFactory` and not `BaseThingHandlerFactory`.